README.md

```diff
@@ -97,6 +97,25 @@ npm run build
 
 Generated files of the website go to the `.next` folder.
 
+### Running the Docker image
+
+To run the Docker image, simply run the following command:
+
+### Steps to start server
+
+- `cd` into the repo
+- run the following command `docker build -t async_image -f Dockerfile . && docker run -d -p 3000:3000 async_image`
+- Visit localhost:3000 and the website should be live
```

Comment:
ok, so we know how to build, and how to run, what about development - this part is critical.

I want to run dev environment for development, so I need to run an image mapping local fork of the repo, right? so I can develop in my IDE, make changes, and rebuild and refresh of website is still possible - through docker this time